### PR TITLE
update quick start demo

### PR DESCRIPTION
**Pull request by 0xpulkit**

### CODE REVIEW

File: docs/quick-start-demo.md [hunk 1]
```diff
@@ -118,88 +102,98 @@ With the new schema in hand, the issuer should now be able to generate a credent
...
 <div align="center">
-    <img width="600" src={useBaseUrl("img/quick-start-demo/verifier-validation.png")}></img>
+    <img width="900" src={useBaseUrl("img/quick-start-demo/wallet-app-verification.png")}></img>
 </div>
 
-:::info
+
+<br></br>
 
 This was a quick demonstration of Privado ID's basic functionalities. However, Privado ID is far more complex than this. It offers a range of SSI-focused tools that allow for decentralized identity and verifiable credentials management.
```
Comment (amonsosanz) [inline docs/quick-start-demo.md:197]: I would change this ending paragraph:

> This quick-start guide demonstrates Privado ID’s basic functionalities through a POAP use case, covering wallet setup, credential issuance, retrieval, and verification. While this is a simplified example, Privado ID provides a comprehensive suite of SSI tools for managing decentralized identity and verifiable credentials.

File: docs/quick-start-demo.md [hunk 2]
```diff
@@ -23,57 +23,41 @@ This guide will briefly touch on the 3 roles of the [Triangle of Trust](introduc
...
-To get started with the Polygon ID Wallet, download the Polygon ID Wallet App and create an Identity:
-
-- For Android: <a href="https://play.google.com/store/apps/details?id=com.polygonid.wallet" target="_blank">Polygon ID on Google Play</a>
-- For iOS: <a href="https://apps.apple.com/us/app/polygon-id/id1629870183" target="_blank">Polygon ID on the App Store</a>
-
+To store verifiable credentials, the Identity Holder (the individual receiving the credential) needs a compatible wallet. Here, we’ll use the Privado ID Wallet, which supports both mobile and web-based access.
 :::note
 
-Polygon ID wallet is an implementation of the Wallet SDK, as a way of showcasing its possibilities. Head to [<ins>the Wallet SDK documentation</ins>](./wallet/wallet-sdk/polygonid-sdk/polygonid-sdk-overview.md) to know more about how it works.
+You can also use any Privado ID compatible wallet. Please, check our [<ins>Ecosystem page</ins>](https://marketplace.polygonid.me/ecosystem) for other options.
```
[the remark this answers is not in the document]
Comment (0xpulkit) [reply docs/quick-start-demo.md]: Fixed

File: docs/quick-start-demo.md [hunk 3]
```diff
@@ -23,57 +23,41 @@ This guide will briefly touch on the 3 roles of the [Triangle of Trust](introduc
...
+- Privado ID Wallet App is an implementation of the [<ins>Wallet SDK</ins>](/docs/category/wallet-sdk), as a way of showcasing its possibilities. Learn more about the Wallet App [<ins>here</ins>](/docs/wallet/wallet-app/privadoid-app.md).
 :::
 
 ## Issue a new credential to attest to the ID Holder's event attendance
 
 A trusted entity, for instance, a private institution will now play the role of an issuer. It will be responsible for creating the credential and sending it to the ID Holder.
 
-We are using <a href="https://user-ui:password-ui@issuer-ui.polygonid.me">the Issuer Node UI testing environment</a> to manage credentials. This is the place where the trusted entity can create credentials, manage schemas and generate connections.
+We are using <a href="https://issuer-demo.privado.id/">the Issuer Node UI testing environment</a> to manage credentials. This is the place where you as an issuer can can create and manage identities and credentials, generate connections and manage schemas.
```
[the remark this answers is not in the document]
Comment (0xpulkit) [reply docs/quick-start-demo.md]: Fixed

File: docs/quick-start-demo.md [hunk 4]
```diff
@@ -23,57 +23,41 @@ This guide will briefly touch on the 3 roles of the [Triangle of Trust](introduc
 
 These are the steps we will cover in this article:
 
-1. [Set up a Polygon ID wallet](#set-up-a-polygon-id-wallet)
+1. [Set up a Privado ID wallet](#set-up-a-polygon-id-wallet)
 2. [Issue a new credential to attest to the ID Holder's attendance to the event](#issue-a-new-credential-to-attest-to-the-id-holders-event-attendance)
 3. [Fetch the newly created credential](#fetch-the-newly-created-credential)
 4. [Verify the credential validity](#verify-the-id-holder-credential)
```
[the remark this answers is not in the document]
Comment (0xpulkit) [reply docs/quick-start-demo.md]: Done

File: docs/quick-start-demo.md [hunk 5]
```diff
@@ -118,88 +102,98 @@ With the new schema in hand, the issuer should now be able to generate a credent
...
 <div align="center">
-    <img width="300" src={useBaseUrl("img/quick-start-demo/quick-fetch-6.jpg")}></img>
+    <img src={useBaseUrl("img/quick-start-demo/wallet-app.png")}></img>
 </div>
 
+
+
 ## Verify the ID holder credential
 
 Here comes the third role in this tutorial: the verifier. This could be represented by an organization that needs to check the accuracy of someone's credentials. In our use case, this organization wants to verify whether the ID holder actually attended our made-up Paris event.
```
[the remark this answers is not in the document]
Comment (0xpulkit) [reply docs/quick-start-demo.md]: Fixed

File: docs/quick-start-demo.md [hunk 6]
```diff
@@ -118,88 +102,98 @@ With the new schema in hand, the issuer should now be able to generate a credent
...
 
 Here are the steps to verify the credential:
 
-1. Visit the [Query builder website](https://schema-builder.polygonid.me/query-builder/). The [Query Builder](/docs/verifier/query-builder/) is an awesome tool designed to simplify the creation of verification queries.
+1. Visit the [Query builder website](https://tools.privado.id/query-builder/). The [Query Builder](/docs/verifier/query-builder/) is an awesome tool designed to simplify the creation of verification queries.
```
[the remark this answers is not in the document]
Comment (0xpulkit) [reply docs/quick-start-demo.md]: Removed

File: docs/quick-start-demo.md [hunk 7]
```diff
@@ -118,88 +102,98 @@ With the new schema in hand, the issuer should now be able to generate a credent
...
 <div align="center">
     <img width="600" src={useBaseUrl("img/quick-start-demo/verifier-home.png")}></img>
 </div>
 
 2. You now need to define the query. You will again make use of the JSON-LD URL which we have also provided: `ipfs://QmdH1Vu79p2NcZLFbHxzJnLuUHJiMZnBeT7SNpLaqK7k9X`. Here is how the query should look like:
 
-<div align="center">
+<div align="center">v
     <img width="600" src={useBaseUrl("img/quick-start-demo/verifier-query-1.png")}></img>
     <img width="600" src={useBaseUrl("img/quick-start-demo/verifier-query-2.png")}></img>
```
[the remark this answers is not in the document]
Comment (0xpulkit) [reply docs/quick-start-demo.md]: Thanks Alex, have added a single one

File: docs/quick-start-demo.md [hunk 8]
```diff
@@ -118,88 +102,98 @@ With the new schema in hand, the issuer should now be able to generate a credent
...
+4. Click **Sign in** and connect your crypto wallet. As you have already claimed the credential, it shows as 'claimed'. Click **Verify**, after which the process of generating the proof starts.
 
 <div align="center">
-    <img width="300" src={useBaseUrl("img/quick-start-demo/verifier-request.jpg")}></img>
+    <img width="900" src={useBaseUrl("img/quick-start-demo/web-wallet-verification.png")}></img>
 </div>
 
-5. Click on Approve. After which, the process of generating the proof starts:
-
+5. And finally, the proof is generated. The verifier will check the revocation status and some additional information. The proof is then sent and validated by the verifier. You will receive the following response on the Query builder website:
```
[the remark this answers is not in the document]
Comment (0xpulkit) [reply docs/quick-start-demo.md]: Fixed

File: docs/quick-start-demo.md (hunk 1 above)
Comment (0xpulkit) [reply docs/quick-start-demo.md]: fixed

File: docs/quick-start-demo.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (amonsosanz) [reply docs/quick-start-demo.md]: I still see mismatches

File: docs/quick-start-demo.md [hunk 9]
```diff
@@ -118,88 +102,96 @@ With the new schema in hand, the issuer should now be able to generate a credent
...
+
 ## Verify the ID holder credential
 
-Here comes the third role in this tutorial: the verifier. This could be represented by an organization that needs to check the accuracy of someone's credentials. In our use case, this organization wants to verify whether the ID holder actually attended our made-up Paris event.
+Here comes the third role in this tutorial: the verifier. This could be represented by an organization that needs to verify some details of someone's credentials. In our use case, this organization wants to verify whether the ID holder actually attended our made-up Paris event.
 
 Here are the steps to verify the credential:
 
-1. Visit the [Query builder website](https://schema-builder.polygonid.me/query-builder/). The [Query Builder](/docs/verifier/query-builder/) is an awesome tool designed to simplify the creation of verification queries.
+1. Visit the [Query builder website](https://tools.privado.id/query-builder/). The [Query Builder](/docs/verifier/query-builder/) is an tool designed to simplify the creation of verification queries.
```
Comment (amonsosanz) [inline docs/quick-start-demo.md:155]: `a tool`

File: docs/quick-start-demo.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (0xpulkit) [reply docs/quick-start-demo.md]: Okay the second one

[the remark this answers is not in the document]
Comment (0xpulkit) [reply docs/quick-start-demo.md]: fixed

**Pull request merged**